src/Promitor.Core.Scraping/Configuration/Model/Metrics/ResourceTypes/ContainerRegistryResourceDefinition.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Promitor.Core.Scraping.Configuration.Model.Metrics.ResourceTypes
+{
+    public class ContainerRegistryResourceDefinition : AzureResourceDefinition
+    {
+        public ContainerRegistryResourceDefinition() : base(ResourceType.ContainerRegistry)
```

Comment:
They actually aren't needed - I'd left them in because there was a `new()` type constraint on the `Scraper` class, but it doesn't look like that's required, so I've removed the constraint and the empty constructors now.

I had to add a custom mapping in for the `ServiceBusQueueMetricDefinitionV1 -> ServiceBusQueueResourceDefinition` mapping because the parameter name in the constructor is `ns` which doesn't match the property name, `Namespace`. Just for the record, the catch all test in the `AutoMapperTests` caught this, so I didn't add an extra test for it.